talawa_lint/pubspec.yaml

```
@@ -1,21 +1,21 @@
   lint: ^2.3.0
+  args: ^2.4.2
 
 dependencies:
   # we will use analyzer for inspecting Dart files
-  # _fe_analyzer_shared: ^61.0.0
   analyzer: ">=5.12.0 <=7.3.0"
-  analyzer_plugin: ^0.11.0
-  # args:
+  analyzer_plugin: ^0.13.0
+  # we will use analyzer for inspecting Dart files
+  analyzer: ">=5.12.0 <=7.3.0"
```

YAMLlint (1.37.1)

[error] duplication of key "analyzer" in mapping

(key-duplicates)